src/OrleansRuntime/Startup/IStartup.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Runtime.MembershipService;
+using Orleans.Runtime.ReminderService;
+
+namespace Orleans.Runtime.Startup
+{
+    public interface IStartup
```

Comment:
Fair point.  At first glance it looked like the app builder could be used to build subsystems like streaming or storage, but upon further inspection, yeah, it's clearly for building http request delegates.